modules/n1ql/pages/n1ql-language-reference/reservedwords.adoc

```diff
@@ -171,133 +171,145 @@ The following keywords are reserved and cannot be used as unescaped identifiers:
 | OBJECT
 | xref:n1ql-language-reference/offset.adoc[OFFSET]
-
 | ON
 | OPTION
 | xref:n1ql-language-reference/insert.adoc#insert-values[OPTIONS]
+
 | xref:n1ql-language-reference/logicalops.adoc#or-operator[OR]
 | xref:n1ql-language-reference/orderby.adoc[ORDER]
 | xref:n1ql-language-reference/window.adoc#window-frame-exclusion[OTHERS]
-
 | OUTER
 | xref:n1ql-language-reference/window.adoc[OVER]
 | PARSE
+
 | PARTITION
 | PASSWORD
 | PATH
-
 | POOL
 | xref:n1ql-language-reference/window.adoc#window-frame-extent[PRECEDING]
-| PREPARE
+| xref:n1ql-language-reference/prepare.adoc[PREPARE]
+
 | PREV
 | PREVIOUS
 | PREVVAL
-
 | PRIMARY
 | PRIVATE
 | PRIVILEGE
+
 | xref:n1ql-language-reference/join.adoc#use-hash-hint[PROBE]
 | PROCEDURE
 | PUBLIC
-
 | xref:n1ql-language-reference/window.adoc#window-frame-clause[RANGE]
 | RAW
 | READ
+
 | REALM
 | RECURSIVE
 | REDUCE
-
 | RENAME
 | REPLACE
 | xref:n1ql-language-reference/window.adoc#nulls-treatment[RESPECT]
+
 | RESTART
 | RESTRICT
 | RETURN
-
 | RETURNING
-| REVOKE
+| xref:n1ql-language-reference/revoke.adoc[REVOKE]
 | RIGHT
+
 | ROLE
+| ROLES
 | xref:n1ql:n1ql-language-reference/rollback-transaction.adoc[ROLLBACK]
 | xref:n1ql-language-reference/window.adoc#window-frame-extent[ROW]
-
 | xref:n1ql-language-reference/window.adoc#window-frame-clause[ROWS]
 | SATISFIES
+
 | xref:n1ql:n1ql-language-reference/savepoint.adoc[SAVEPOINT]
 | SCHEMA
 | SCOPE
 | xref:n1ql-language-reference/selectclause.adoc[SELECT]
-
 | SELF
 | SEQUENCE
+
+| SEMI
 | SET
 | SHOW
 | SOME
 | START
-
 | STATISTICS
+
 | STRING
 | SYSTEM
 | THEN
 | xref:n1ql-language-reference/window.adoc#window-frame-exclusion[TIES]
 | TO
-
 | xref:n1ql:n1ql-language-reference/begin-transaction.adoc[TRAN]
+
 | xref:n1ql:n1ql-language-reference/begin-transaction.adoc[TRANSACTION]
 | TRIGGER
 | TRUE
 | TRUNCATE
 | xref:n1ql-language-reference/window.adoc#window-frame-extent[UNBOUNDED]
-
 | UNDER
+
 | xref:n1ql-language-reference/union.adoc[UNION]
 | UNIQUE
 | UNKNOWN
 | xref:n1ql-language-reference/unnest.adoc[UNNEST]
-| UNSET
-
+| xref:n1ql-language-reference/update.adoc#unset-clause[UNSET]
 | xref:n1ql-language-reference/update.adoc[UPDATE]
+
 | xref:n1ql-language-reference/upsert.adoc[UPSERT]
 | xref:n1ql-language-reference/hints.adoc[USE]
 | USER
 | USERS
 | USING
-
 | VALIDATE
+
 | VALUE
 | VALUED
 | VALUES
 | VECTOR
 | VIA
-
 | VIEW
+
 | WHEN
 | xref:n1ql-language-reference/where.adoc[WHERE]
 | WHILE
-| WINDOW
-| WITH
-
+| xref:n1ql-language-reference/window.adoc[WINDOW]
+| xref:n1ql-language-reference/with.adoc[WITH]
 | xref:n1ql-language-reference/collectionops.adoc#collection-op-within[WITHIN]
+
 | xref:n1ql:n1ql-language-reference/begin-transaction.adoc[WORK]
 | XOR
 |
 |
 |
+|
+
 |===
+
+TIP: You can combine `USING` with the keyword `AI` to form the statement `USING AI`.
+This statement uses AI capabilities to convert natural language prompts into {sqlpp} queries.
+For more information, see xref:n1ql-language-reference/using-ai.adoc[].
```

Comment:
My point is that `AI` is not a reserved word, even though there is a `USING AI` statement

```suggestion
NOTE: The word `AI` is not a reserved word, even though it can be used in combination with the `USING` keyword as part of the  xref:n1ql-language-reference/using-ai.adoc[USING AI] statement.
You do not have to escape the word `AI` inside backticks when using it by itself as an identifier.
```